kontrakter/politi/varetekt/1.0/begjaeringVaretekt.schema.json

```diff
@@ -1191,6 +1213,18 @@
       },
       "required": ["internId"],
       "additionalProperties": false
+    },
+    "straffesaksnummer": {
+      "type": "string",
+      "description": "Som regel et løpenummer, men kan i fremtiden også inneholde bokstander",
+      "minLength": 3,
+      "maxLenght": 30
+    },
+    "telefonnummer": {
+      "description": "Ett telefonnummer, norsk eller utenlands, med eller uten land prefix",
+      "maxLength": 30,
+      "minLength": 1,
+      "type": "string"
```

Comment:
`"type": "string"` øverst for å være konsekvent med andre definisjoner

Reply:
Ryddet